build.gradle.kts

```diff
@@ -260,6 +260,7 @@ tasks {
 				dependencies {
 					implementation(project)
 					implementation("com.fasterxml.jackson.core:jackson-databind:$jacksonVersion")
+					implementation("org.assertj:assertj-core:3.22.0")
```

Comment:
This seems not to work as `assertThat` is not found during compile (but by IntelliJ) in demo sources